bylaws.tex

```diff
@@ -161,6 +161,9 @@
 \bsubsubsection{Introductory Project Advisors} 
 The Evaluations Director may choose between one (1) and three (3) House members to be Introductory Project Advisors.
 The advisors assist in organizing the participants at the beginning of the Introductory Project and thereafter act as a resource to keep the Project running smoothly.
+\bsubsubsection{Introductory Project Evaluations Process}
+The Evaluations Director determines whether each introductory member passes or fails the Introductory Project by using information gathered from the Introductory Project Executive Board.
```

Comment:
Giving 4 freshmen an opportunity to show that their grown ups isn't worth the risk of them being completely incapable of telling their peers about their failures.